### DISCUSSION TITLE
Are there any plans to support cloud-run jobs or CloudFunctions deployments?

### DISCUSSION THREAD

**Discussion by SakagamiKazuto** [pipe-cd/pipecd#4778] [Q&A]
Are there any plans to support Cloudrun job or CloudFunctions deployments?

When using GCP, services often handle Cloudrun jobs and Cloud functions in addition to Cloudrun services, so we would like to confirm whether these deployments are supported.

**Comment by khanhtc1202** [comment]
Hi @SakagamiKazuto, thanks for the question. Currently, we're implementing the pluginable arch for the piped and will release it in the next few versions (the plan is set to April). After that, we can add other Application Kinds which can be managed by piped more effectively. Is that good enough for your case 👀

**Comment by t-kikuc** [comment]
We'll support Cloud Run jobs as a plugin:

https://github.com/pipe-cd/pipecd/issues/5812

I've forgotten this discussion...
